stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Two narratives, technical side in the last paragraph (this is just a note for when we publish to change the color of it).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>The link for update in over 50s being close to 95% relates to my concerns about how full vaccination is presented above. Wondering if this should be introduced into the high level narrative earlier.</p>